ui/apps/platform/src/Containers/Clusters/InitBundles/SecureClusterUsingOperator.tsx

```diff
@@ -8,16 +12,93 @@ export type SecureClusterUsingOperatorProps = {
 function SecureClusterUsingOperator({
     headingLevel,
 }: SecureClusterUsingOperatorProps): ReactElement {
+    const { version } = useMetadata();
+    const subHeadingLevel = headingLevel === 'h2' ? 'h3' : 'h4';
+
     return (
         <Flex direction={{ default: 'column' }}>
             <Title headingLevel={headingLevel}>
                 Secure a cluster using Operator installation method
             </Title>
+            {version && (
+                <ExternalLink>
+                    <a
+                        href={getVersionedDocs(
+                            version,
+                            'installing/installing_ocp/install-secured-cluster-ocp.html'
+                        )}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Installing secured cluster services on Red Hat OpenShift
+                    </a>
+                </ExternalLink>
+            )}
+            <p>
+                You can install secured cluster services on your clusters by using the{' '}
+                <strong>SecuredCluster</strong> custom resource.
+            </p>
+            <Title headingLevel={subHeadingLevel}>
+                Procedure to do only once before you secure the first cluster
+            </Title>
             <List component="ol">
-                <ListItem>Do something.</ListItem>
-                <ListItem>Do something.</ListItem>
-                <ListItem>Do something.</ListItem>
+                <ListItem>
+                    <p>Use the OpenShift console to install the ACS Operator from Operator Hub.</p>
+                    <p>
+                        Create a new OCP project for ACS. A good name choice is{' '}
+                        <strong>rhacs-operator</strong>
```

Comment:
period needed at end of sentence?

Reply:
Because I sometimes include adjacent punctuation when I copy and paste from sentences, what do you think about rewriting with period as follows:

**rhacs-operator** is a good name choice.

Reply:
Is there a more specific adjective than good?

Reply:
I generally don't like starting a sentence with a lower-case item but it does avoid some awkwardness...your call :)

Reply:
> Is there a more specific adjective than good?

No, I don't think it matters what you use for the operator namespace (whereas it does matter in other areas) so I think good works?